src/Bubble.js

```diff
@@ -52,6 +53,25 @@ export default class Bubble extends React.Component {
     return null;
   }
 
+  renderTicks() {
+    if (this.props.renderTicks) {
+        return this.props.renderTicks(currentMessage);
+    }
+
+    const {currentMessage} = this.props;
```

Comment:
This line should be above the if statement - you are using `currentMessage` and it is not defined